core/services/relay/evm/codec_entry.go

```diff
@@ -33,7 +33,7 @@ func (entry *codecEntry) Init() error {
 	checked := make([]reflect.StructField, argLen)
 
 	if len(args) == 1 && args[0].Name == "" {
-		nativeArg, checkedArg, err := getNativeAndCheckedTypes(&args[0].Type)
```

Comment:
I would add a comment above that says // evm log "without" topics, just event signature

Reply:
this could be a return.  Eg: 

`function foo() returns int256`

is a single element that's unnamed.  Here, we don't want to wrap the `int256` into a `struct`.  But if you have

`function foo() returns (i int256, j int256)` the type returned will be a `struct` with the two fields.

so would 
`function foo() returns (i int256)` because they wanted to name i.

Reply:
ahh, missed that, I think it makes sense to put both of these examples into a comment( log and method).